### PR TITLE
feat/add support php8.1

### PR DESCRIPTION
**Pull request by mvarlic**
* add support to PHP 8.1

image
image

### CODE REVIEW

**Review by tribiec — APPROVED** [review]
lgtm!

File: webpay/src/Helpers/InteractsWithOneclick.php [hunk 1]
```diff
@@ -95,7 +95,7 @@ protected function getCountCardsByUserId($userId)
         return SqlHelper::getValue('SELECT count(1) FROM '._DB_PREFIX_.TransbankInscriptions::TABLE_NAME.' WHERE `status` = "'.TransbankInscriptions::STATUS_COMPLETED.'" and `user_id` = "'.pSQL($userId).'"');
```
Comment (mastudillot) [inline webpay/src/Helpers/InteractsWithOneclick.php:95]: Was this variable escaped?

File: docker-prestashop-php8.1-pres8.0.5-apache/README.md
Comment (mastudillot) [inline docker-prestashop-php8.1-pres8.0.5-apache/README.md:1]: this change is a doc not chore

Comment (mastudillot) [reply docker-prestashop-php8.1-pres8.0.5-apache/README.md:1]: the README file should go in a 'doc' type commit, separate from the 'chore'

**Comment by mastudillot** [comment]
How do you use the example composer files in the plugin?

**Pull request closed without merging**